classes/Application/Speakers.php

```diff
@@ -63,7 +63,7 @@ public function findProfile()
      * @return Talk
      * @throws NotAuthorizedException
      */
-    public function getTalk($talkId)
+    public function getTalk(int $talkId)
```

Comment:
I'm not entirely sure whether we have filtering in place that sorts out passing in a possibly *integerish* `string` value via the UI, which if we didn't would fail with a `TypeError`. What do you think, do we need to add missing tests here?

Also, what do think about adding/updating the corresponding `@param`s in the docblocks?

Reply:
I checked where this function is called, and its only used in one place (outside of automated tests), which is the talk view action. And there it gets filtered to an int anyways.

So if we would call this from another place in the future it should enforce some validation as well.